config/envs/development.py

```diff
@@ -25,7 +25,7 @@ class DevelopmentConfig(DefaultConfig):
     SSO_LOGIN_URL = AUTHENTICATOR_HOST + "/sso/login"
     SSO_LOGOUT_URL = AUTHENTICATOR_HOST + "/sso/logout"
 
-    DEBUG_USER_ON = False  # Set to True to use DEBUG user
+    DEBUG_USER_ON = True  # Set to True to use DEBUG user
```

Comment:
woops gonna revert this

Reply:
https://github.com/communitiesuk/funding-service-design-assessment/pull/235/commits/5e14b6f8c85e51a51eb467db664f147905a2fa8b